src/UI/ADnoteUI.fl

```diff
@@ -190,10 +190,6 @@ adnoteui->ADnoteVoice->show();
 adnoteui->currentvoicecounter->value(nvoice+1);
 adnoteui->currentvoicecounter->do_callback();
 class ADvoiceUI *advoice = adnoteui->advoice;
```

Comment:
This leaves an unused variable `advoice`, right?

Reply:
removed it.